lib/core/src/sdk.rs

```diff
@@ -1243,19 +1253,34 @@ impl LiquidSdk {
                 self.pay_liquid(liquid_address_data.clone(), amount_sat, *fees_sat, true)
                     .await
             }
-            SendDestination::Bolt11 { invoice } => {
+            SendDestination::Bolt11 { invoice, .. } => {
                 self.pay_bolt11_invoice(&invoice.bolt11, *fees_sat).await
```

Comment:
Would it be an option to insert the payment details with the `bip353_address` here (if set), then we don't have to deal with extracting the `bip353_address` from the SendDestination in `lnurl_pay()`?

Reply:
Good idea! [10b31e3](https://github.com/breez/breez-sdk-liquid/pull/718/commits/10b31e3970c6aceb8fc3a8a1322a0e502c7bf722)